src/components/HomePage.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable react/prop-types */
+// src/components/HomePage.js
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import Filter from './Filter';
+import { fetchCoordinates, selectState } from '../actions';
+
+const HomePage = ({
+  items, loading, error, fetchCoordinates, selectState,
+}) => {
+  if (loading) return <div>Loading...</div>;
+  if (error) {
+    return (
+      <div>
+        Error:
+        {' '}
+        {error}
+      </div>
+    );
+  }
+
+  return (
+    <div>
+      <Filter fetchCoordinates={fetchCoordinates} selectState={selectState} />
+      <ul>
+        {Array.isArray(items.list) ? items.list.map((item, index) => (
+          // eslint-disable-next-line react/no-array-index-key
```

Comment:
- [ ]  kindly ensure that no linter is disabled, linters are tools that analyze your code to identify and flag potential issues, errors, or stylistic inconsistencies. They are widely used in software development to ensure code quality and maintainability. so kindly try to solve the issue not disable it 💯💪🏻⭐ I am sure you will find a solution for it 👏🏻💪🏻💯